Assets/Game/Addons/ModSupport/ModManager.cs

```diff
@@ -370,6 +370,19 @@ public static string GetAssetName(string assetPath)
             return assetPath.Substring(startIndex).ToLower();
         }
 
+        /// <summary>
+        /// Utility method for gettings lines of text from a text asset
+        /// </summary>
+        public static List<string> GetTextAssetLines(TextAsset asset)
+        {
+            StringReader reader = new StringReader(asset.text);
```

Comment:
Just a very small note: StringReader implements IDisposable so you probably want a using block.

Reply:
Another instance where I prove I am still transitioning from Java to C#. Great to have stuff like this pointed out - thank you. :-)